hi-ml-cpath/testhisto/testhisto/mocks/tiles_generator.py

```diff
@@ -114,7 +114,7 @@ def generate_mock_histo_data(self) -> None:
                 raise NotImplementedError
 
             tile_filename = self.dest_data_path / row[PandaTilesDataset.IMAGE_COLUMN]
-            save_image(tile.float(), tile_filename)
```

Comment:
save_image does some funky rescaling (assuming that the image is in [0, 1] scale )which was resulting in just white tiles
<img width="412" alt="image" src="https://user-images.githubusercontent.com/37396332/184619166-bb064254-8bea-4512-b519-a2ff8e67ce0c.png">